NineChronicles.Headless/MemoryCacheExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Bencodex;
+using Bencodex.Types;
+using MessagePack;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace NineChronicles.Headless;
+
+public static class MemoryCacheExtensions
+{
+    private static readonly Codec Codec = new Codec();
+    private static readonly MessagePackSerializerOptions Lz4Options = MessagePackSerializerOptions.Standard.WithCompression(MessagePackCompression.Lz4BlockArray);
+
+    public static byte[] SetSheet(this MemoryCache cache, string cacheKey, IValue value, TimeSpan ex)
+    {
+        var compressed = MessagePackSerializer.Serialize(Codec.Encode(value), Lz4Options);
+        cache.Set(cacheKey, compressed, ex);
+        return compressed;
+    }
+
+    public static bool TryGetSheet<T>(this MemoryCache cache, string cacheKey, out T cached)
+    {
+        return cache.TryGetValue(cacheKey, out cached);
+    }
```

Comment:
(This is a review you can skip if you’re in a hurry) Is there a reason to make an alias of `cache.TryGetValue`?

Reply:
After modifying the code, it no longer need it. I will delete this.